feat(COC): add COC script tags data source (#7764)
huaweicloud/terraform-provider-huaweicloud: docs/data-sources/coc_script_tags.md

---
subcategory: "Cloud Operations Center (COC)"
layout: "huaweicloud"
page_title: "HuaweiCloud: huaweicloud_coc_script_tags"
description: |-
  Use this data source to get the list of COC script resource tags.
---

# huaweicloud_coc_script_tags

Use this data source to get the list of COC script resource tags.

## Example Usage

```hcl
data "huaweicloud_coc_script_tags" "test" {
  resource_type = "coc:script"
}
```

## Argument Reference

The following arguments are supported:

> * `resource_type` - (Required, String) Specifies the resource type. Generally, coc-script only uses **coc:script**.

Reviewer: ```suggestion
* `resource_type` - (Required, String) Specifies the resource type. Valid value is **coc:script**.
```
